app/scripts/views/mixins/resume-token-mixin.js

```diff
@@ -21,10 +21,15 @@ define([
       // there might not be any relier if the resume token is being fetched
       // for an account unlock request caused by changing the password.
       var relierInfo = this.relier && this.relier.pickResumeTokenInfo();
+      var userInfo = this.user && this.user.pickResumeTokenInfo();
 
       // When account or user fields are needed,
       // they can be added as part of the resumeTokenInfo
-      var resumeTokenInfo = _.extend({}, relierInfo);
+      var resumeTokenInfo = _.extend(
+        {},
+        relierInfo,
+        userInfo
+      );
```

Comment:
Good point. We could also do something like:

```js
var resumeTokenInfo = {
  relier: relierInfo,
  user: userInfo
};
```

We'll have to accommodate/handle already existing resume tokens. We could take the simple approach and just add a `user` field to the existing data structure:

```js
var resumeTokenInfo = _.extend(
  {},
  relierInfo,
  user: userInfo
})
```

Or we could attempt to manually update the old style resume token to the new style.

Do we have to start versioning resume tokens?